plugins/CoreHome/CoreHome.php

```diff
@@ -75,9 +75,14 @@ public function addStylesheets(&$mergedContent)
 
     public function initAuthenticationObject()
```

Comment:
This unrelated to this PR, but this function name confused me, combined w/ the `return;` confused me. It might be better to rename it to something like `checkAllowedIpsOnAuthentication` or something.